packages/core/src/amazonq/lsp/lspClient.ts

```diff
@@ -259,6 +266,7 @@ export async function activate(extensionContext: ExtensionContext, resourcePaths
         debug: { module: serverModule, transport: TransportKind.ipc, options: debugOptions },
     }
 
+    // TODO(jmkeyes): this overwrites the above...?
     serverOptions = createServerOptions({
```

Comment:
The `let serverOptions = ...` immediately above this line is not used. Is this a bug?

Reply:
yeah it overwrites above, there's no reason for the first serverOptions to exist